### PR TITLE
Doc 59 - adding zksynv v2 FAQ

### PR DESCRIPTION
**Pull request by ramonatmatterlabs**

### CODE REVIEW

File: docs/dev/fundamentals/faq.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    - zkVM backend.
+- Special-purpose circuits (heavily relying on PLONK’s custom gates and lookup tables) as “precompiles” for computationally intensive operations, such as:
+    - Non-algebraic hashes (Keccak, SHA256, Blake2).
+    - Storage access (Merkle paths).
+    - Elliptic curve pairing.
+- Recursive aggregation circuit (combines the proofs from the above-mentioned parts).
+
+### zkEVM vs EVM
+
+Apart from the opcodes and gas metering disparity, zkVM strictly inherits the EVM programming model and its invariants, including the ABI calling conventions. One important thing to emphasize is that the zkVM supports rollbacks and provably revertible transactions. It guarantees mutual protection: users can not stall the network with bombardment by revertible transactions, and the escape hatch (priority queue) protect the user’s ability to include any transactions into the blocks.
```
Comment (uF4No) [inline docs/dev/fundamentals/faq.md:42]: In this paragraph we mention zkVM but the title is zkEVM.

**Comment by ramonatmatterlabs** [comment]
There was a discussion on notion, and Filip and Anthony decided to leave
like this for now

File: docs/dev/fundamentals/faq.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+### EVM Improvements
+
+While maintaining maximum compatibility, the zkEVM has significant improvements over the EVM that increase adoption and benefit our ecosystem partners.
+
+- **Our compiler is based on LLVM**. LLVM-based compilers (Low-Level Virtual Machine) have become the default compiler for Mac OS X, iOS, FreeBSD, and Android systems and are among the most widely used because they:
+    - Enable us to improve the efficiency over the original EVM bytecode because with LLVM we can take advantage of the many optimizations and tools available in this mature ecosystem.
+    - Pave the way for us to add support for integrating codebases written in other programming languages with LLVM frontend. By doing so, developers can build dApps and use blockchains in ways that are currently not possible.
+- **Account Abstraction is included in our zkEVM**. This is a long-anticipated feature in the Ethereum dev community which improves developer adoption and user experience in a number of ways:
+    - Native support for smart contracts wallets (like Argent), which is critical for onboarding mainstream users.
```
Comment (uF4No) [inline docs/dev/fundamentals/faq.md:53]: Maybe add a link to the account abstraction section of the docs

File: docs/dev/fundamentals/faq.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    - Much better UX for multisigs.
+    - Transaction fees can be paid in any token.
```
Comment (uF4No) [inline docs/dev/fundamentals/faq.md:55]: Transaction fees can be paid in any token + "using paymasters" with a link to the paymasters section of the docs.

File: docs/dev/fundamentals/faq.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+- The ability to natively pay gas fees in tokens other than ETH.
+- The ability for accounts to change public and private keys.
+- The addition of non-cryptographic modifications, where users can require transactions to have expiry times, confirm slightly out-of-order, and more.
+- Diversity in signature verification systems from the current ECDSA, including post-quantum safe signature algorithms (eg. Lamport, Winternitz).
+
+In other words, Account Abstraction brings about major improvements to the overall user experience, and expands the application design space for developers. Learn more in [this blog post](https://www.argent.xyz/blog/wtf-is-account-abstraction/) by Argent.
+
+In zkSync 2.0 Account Abstraction is natively implemented, meaning accounts can initiate transactions, like an EOA, but can also have arbitrary logic implemented in them, like a smart contract. 
+
+If you want to better understand what Account Abstraction on zkSync looks like, you can try out our tutorial [here](../tutorials/custom-aa-tutorial.html).
```
Comment (uF4No) [inline docs/dev/fundamentals/faq.md:105]: Also include a link to the account abstraction section of the docs

File: docs/dev/fundamentals/faq.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+**Testnet**
+
+- Network Name: `zkSync alpha testnet`
+- RPC URL: `https://zksync2-mainnet.zksync.io`
+- Chain ID: `280`
+- Currency Symbol: `ETH`
+<!-- - Block Explorer URL: `https://testnet.explorer.zksync.io/` -->
+
+**Mainnet**
```
Comment (uF4No) [inline docs/dev/fundamentals/faq.md:143]: Do we want to publicly share mainnet details?

**Review by uF4No — APPROVED** [review]
approved

**Pull request merged**